test/unit/screenShareController_spec.js

```diff
@@ -145,14 +143,15 @@ describe('ScreenShareController', () => {
   it('should respond correctly to screenShareView:installExtension when ' +
      'installation does not work', (done) => {
     var event = new CustomEvent('screenShareView:installExtension');
-    window.chrome.isGoingToWork = false;
+    var realWindowOpen = window.open;
+    window.open = function () { return null };
```

Comment:
linter doesn't seem to like this. I think it wants to end the return statement with a semicolon. window.open = function () { return null; };